### ISSUE TITLE
DPH u OSVČ

### ISSUE THREAD

**Issue by inson1** [zakjan/czech-income-tax-calculator#26]
Pod 2 miliony možnost opt in, po 2 milionech možnost povinná, ale check box bych tam nechal jen se nebude moc dát změnit

**Comment by zakjan** [comment]
Ano, ale co by ten checkbox měl ovlivňovat? Pokud je plátce DH, pak fakturuje s DPH, to DPH odvede, a pro účely daně z příjmů a odvodů (cíl téhle kalkulačky) bude výsledek stejný.

**Comment by zakjan** [comment]
Maximálně bychom to mohli udělat stejně jako v #24, zobrazit upozornění.
